pkgs/applications/misc/dwmblocks/default.nix

```
@@ -0,0 +1,34 @@
+{ lib, stdenv, fetchFromGitHub, libX11, patches ? [ ], writeText, conf ? null }:
+
+stdenv.mkDerivation {
+  name = "dwmblocks";
```

```suggestion
  pname = "dwmblocks";
  version = "unstable-XXXX-XX-XX";
```

please replace `XXXX-XX-XX` with the date of the commit.